common/src/dodeca.rs

```diff
@@ -43,6 +43,8 @@ impl Side {
         ADJACENT[self as usize][other as usize]
     }
 
+    /// All verticies incident on self
```

Comment:
A few minor typos:
* On this line, "verticies" should be "vertices".
* On line 258, `incedent" should be "incident".
* In math.rs on line 108, "repect" should be "respect".